store/docs/soap-admin.txt

```diff
@@ -3168,3 +3168,34 @@ Corresponding API for outgoing/sent email filters (message structure same as abo
 </ModifyOutgoingFilterRulesRequest>
 
 <ModifyOutgoingFilterRulesResponse/>
+------------------------------
+ContactBackup API
+------------------------------
+<ContactBackupRequest <op="start|stop">>
+    [<servers>
+     <ServerSelector>...</ServerSelector>
+     <ServerSelector>...</ServerSelector>
+     ...
+     </servers>
+    ]
+</ContactBackupRequest>
+
+op : It's a mandatory attribute. Value should be either "start" or "stop"
+start : start contact backup
+stop : stop contact backup
+servers : optional wrapper element for ServerSelectors. if not mentioned all the mailboxe servers will be processed.
+
+<ContactBackupResponse>
+     <servers>
+         [<server name="" status="started|error|stopped"></server>
+         ...
+         ]
+     </server>
+</ContactBackupResponse>
+
+servers : optional wrapper element for servers. if not mentioned, no mailboxe server is processed.
```

Comment:
Suggest "If not mentioned, no backups were done on any mailbox servers"

Reply:
done